packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.js

```diff
@@ -51,7 +52,7 @@ export default class TooltipPortalMounter extends React.Component<Props> {
     }
 
     _getPortalParent() {
-        const anchorNode = ReactDOM.findDOMNode(this);
+        const anchorNode = this.props.anchorNode; //  ReactDOM.findDOMNode(this);
```

Comment:
This should still be a DOM node look up as the anchor could be anything including a component, hence why it was as it was before.

Reply:
The second lookup shouldn't be necessary anymore since the anchor is passed directly as a child to TooltipAnchor now instead of via children-as-function.